### PR TITLE
9544 blocked cases consolidated

### PR DESCRIPTION
**Pull request by rgoshen**

### CODE REVIEW

File: web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  petitionsClerkCreatesNewCase(cerebralTest, fakeFile, trialLocation);
+
+  it('should set member case docket number', () => {
+    memberCaseDocketNumber = cerebralTest.docketNumber;
+  });
+
+  loginAs(cerebralTest, 'docketclerk@example.com');
+  docketClerkSetsCaseReadyForTrial(cerebralTest);
+
+  loginAs(cerebralTest, 'docketclerk@example.com');
```
Comment (ujahio) [inline web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js]: Is there a need to re-login as a docket clerk again at this moment?

File: web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        expect.objectContaining({
+          consolidatedIconTooltipText: 'Lead case',
+          inConsolidatedGroup: true,
+          inLeadCase: true,
+        }),
+      ]),
+    );
+  });
+
+  loginAs(cerebralTest, 'petitionsclerk@example.com');
```
Comment (ujahio) [inline web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js]: Is there a need to re-login as a petitions clerk again at this moment?

File: web-client/src/presenter/computeds/utilities/addConsolidatedProperties.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line spellcheck/spell-checker
+/**
+ * Adds consolidated case properties to the passed in object.
+ *
+ * @param {object} object - Object to have consolidated case properties added into.
+ * @returns {object} Object, consolidatedIconTooltipText, inConsolidatedGroup, and inLeadCase
+ */
+
+export const addConsolidatedProperties = object => {
```
Comment (ujahio) [inline web-client/src/presenter/computeds/utilities/addConsolidatedProperties.js]: nit-pick: please rename `object` to an appropriate name e e.g consolidatedInfoProps

File: web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import { addConsolidatedProperties } from './addConsolidatedProperties';
+
+describe('addConsolidatedProperties', () => {
+  it('should return the expected object when the message passed in is on the lead case & consolidated group', () => {
```
Comment (ujahio) [inline web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js]: nit-pick: Please change the test description to account for the new properties being added. For example `should add the Lead Case tooltip text and set both inConsolidatedGroup and isLeadCase  to true when docket number and leadDocketNumber are the same`

File: web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      leadDocketNumber: '101-20',
+    });
+    expect(result).toMatchObject({
+      consolidatedIconTooltipText: 'Lead case',
+      inConsolidatedGroup: true,
+      inLeadCase: true,
+    });
+  });
+
+  it('should return the expected object when the message passed in is NOT on the lead case & consolidated group', () => {
```
Comment (ujahio) [inline web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js]: nit-pick: Should this test description read something like `should return consolidated case tooltip text, and set inLeadCase to false, when the docket number and lead docket number are different, but in the same consolidated group`

File: web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+      leadDocketNumber: '101-20',
+    });
+    expect(result).toMatchObject({
+      consolidatedIconTooltipText: 'Consolidated case',
+      inConsolidatedGroup: true,
+      inLeadCase: false,
+    });
+  });
+
+  it('should return the expected object when the message passed in is not part of a consolidated group', () => {
```
Comment (ujahio) [inline web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js]: nit-pick: maybe re-write the test description to read something like this: `should not generate a consolidated text tooltip, and set both inConsolidatedGroup and inLeadCase to false when the lead docket number is undefined`

File: web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js (hunk 1 above)
Comment (rgoshen) [reply web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js]: Not sure why I originally had the extra logins.  They have been removed.  Thx

File: web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js (hunk 2 above)
Comment (rgoshen) [reply web-client/integration-tests/petitionsClerkBlockConsolidatedCasesJourney.test.js]: Not sure why I originally had the extra logins.  They have been removed. thx

File: web-client/src/presenter/computeds/utilities/addConsolidatedProperties.js (hunk 3 above)
Comment (rgoshen) [reply web-client/src/presenter/computeds/utilities/addConsolidatedProperties.js]: Renamed to caseObject

File: web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js (hunk 4 above)
Comment (rgoshen) [reply web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js]: changed

File: web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js (hunk 5 above)
Comment (rgoshen) [reply web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js]: changed

File: web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js (hunk 6 above)
Comment (rgoshen) [reply web-client/src/presenter/computeds/utilities/addConsolidatedProperties.test.js]: changed

**Review by rosiehsmith — APPROVED** [review]

**Review by jimlerza — APPROVED** [review]

**Pull request merged**